src/authoring/components/editors/iframe-control.tsx

```diff
@@ -8,17 +8,20 @@ import "./iframe-control.scss";
 (window as any).DISABLE_FIREBASE_SYNC = true;
 
 const urlParams = new URLSearchParams(window.location.search);
-const iframeBase = urlParams.get("iframeBase") ?? ".";
+// note: .. is used below as iframe.html is in root and authoring is in authoring/
+const iframeBase = urlParams.get("iframeBase") ?? "..";
```

Comment:
Is the decap CMS still being used at this point? Perhaps this change will break it? Or perhaps there have been several changes to this iframe-control which prevent decap from using this control? I don't think we need to keep supporting decap unless we are incrementally rolling this new authoring out.